liblwgeom/liblwgeom_topo.h

```diff
@@ -188,6 +188,7 @@ typedef struct LWT_BE_CALLBACKS_T {
    * @param ids an array of element identifiers
    * @param numelems input/output parameter, pass number of node identifiers
    *                 in the input array, gets number of node in output array.
+   *	TODO: Should be uint64 to match SPI_processed
```

Comment:
We don't need a match, btw, as this module is PostgreSQL agnostic. Possibly we do need a bigger integer, to _allow_ for returning more than 4 million faces

Reply:
True, it's design as platform agnostic, but I've only seen this used in the `topology/` folder which is tightly tied to Postgres.